forms-web-app/src/views/eligibility/planning-department.njk

```diff
@@ -16,6 +49,8 @@
           id: "local-planning-department",
           name: "local-planning-department"
         }) }}
+        {% endif %}
```

Comment:
No need for the `if` / `else` here. 

Use a conditional on the `errorMessage` and use the message from the validation error:

```
          {{ govukInput({
            label: {
              text: "What is the name of the local planning department?",
              classes: "govuk-label--l",
              isPageHeading: true
            },
            id: "local-planning-department",
            name: "local-planning-department",
        errorMessage: errors['local-planning-department'] and {
          text: errors['local-planning-department'].msg
        }
          }) }}
```

Also worth checking your error message matches with the one expected in the word document attached to this ticket: https://pins-ds.atlassian.net/browse/UCD-770